c8/client.py

```diff
@@ -11,6 +13,26 @@
 __all__ = ["C8Client"]
 
 
+class CompressionType(Enum):
```

Comment:
@dlozina-macrometa please check if similar changes are needed in stream_collection.py or not..